config/routes.rb

```diff
@@ -124,6 +116,13 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
         get "link-sent", to: "client_logins#link_sent", as: :login_link_sent, on: :collection
         get "invalid-token", to: "client_logins#invalid_token", as: :invalid_token, on: :collection
       end
+      resources :tax_returns, only: [], path: '/tax-returns' do
```

Comment:
Most of the client-facing routes seem to be dasherized, so I updated these routes to follow that naming convention.